src/components/search/content-highlights/data/utils.js

```diff
@@ -10,8 +10,13 @@ export const getFormattedContentType = (contentType) => {
 };
 
 export const getAuthoringOrganizations = (authoringOrganizations) => {
+  if (!authoringOrganizations) {
```

Comment:
[curious] Under what circumstances is `authoringOrganizations` not defined? Was this added to better handle a test case? I believe `authoringOrganizations` from the API is always an array, even if empty. I suppose it doesn't hurt to handle the case when it doesn't exist just in case though.

Reply:
Yep that was exactly why I added it in. It was failing the empty or nonexistent use case state, so returning an empty string was my solution

Reply:
The alternative solution would be to ensure none of the test data has a nonexistent `authoringOrganizations` to more closely reflect our API responses. Can't hurt to be proactively defensive, either way though.